docs/VMR-Feature-Band-Support-Work-Plan.md

```diff
@@ -9,9 +9,16 @@ Document convention:
 
 ### Branch Management
 
-The 2xx branch sources should accurately reflect what gets produced in the output. Since the shared component are only being produced in the 1xx branch, those associated repos will be excluded from the 2xx branch upon creation.
+The 2xx branch sources should accurately reflect what gets produced in the output. Since the shared component are only being produced in the 1xx branch, those associated repos will be excluded from the 2xx branch upon creation. The benefits of of removing the source include the following:
+* Avoids confusion about which source is relevant in the production of the build
+* Avoids the need to determine if or when repo sources should be updated with the latest from the 1xx branch.
+* Avoids any compliance issues with the code getting stale if it hasn't been updated (e.g. CodeQL alerts).
 
-There should be tooling to help with this exclusion process for the VMR maintainers. Similarly, the associated projects in the `repo-projects` directory should be deleted along with updating the appropriate `RepositoryReference` usages.
+One exception is the Arcade repo which will be present in all branches. It's not technically required to exist in the 2xx branch in all cases. But there have historically been cases where Arcade changes are necessary in order to support new SDK features not in the 1xx branch. To account for those cases, it'll be a smoother process to simply always have Arcade present in those branches. There isn't an impact in the output of the build since Arcade isn't a shipping component.
```

Comment:
(How) will the Arcade be updated? We should also distinguish between root-level Arcade dependency and `src/arcade` and describe their lifecycle.

Reply:
> (How) will the Arcade be updated?

It would be updated in the same way it's updated in the 1xx branch. Is that an issue?

Reply:
> It would be updated in the same way it's updated in the 1xx branch. Is that an issue?

Arcade doesn't snap branches for different bands. So backflow won't work. Will we be forbidden from making changes to it in the 2xx branch?